### DISCUSSION TITLE
Scammy/Spammy bots have made their way to GitHub

### DISCUSSION THREAD

**Discussion by TechStudent10** [community/community#42869] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Posting this so that the GitHub team knows.

#### Note: If anyone stumbles upon these bots, here is what I would recommend you do:
1. Report the comment
2. Block the account
3. Delete the comment

I got a notification recently about a comment on a discussion post of mine. The comment in question is (links blurred for obvious reasons):

![Scam bots on Github (why)](https://user-images.githubusercontent.com/76978184/209984622-61930c01-1500-436a-9aa4-efd06d8e0445.png)

The accounts pinged were mostly created this month.

**Comment by Akash1134** [comment, marked as answer]
Hi @TechStudent10, GitHub is responding to concerns expressed by community members regarding recent spam posts & comments. Thanks for sharing and we appreciate your patience.

**Reply by TechStudent10** [reply]
Ah okay 👍🏾. Good luck to the GitHub team with eradicating these annoying bots!
